src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -209,7 +212,6 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onStart() {
         Log_OC.v(TAG, "onStart() start");
         super.onStart();
-        getSupportActionBar().setIcon(DisplayUtils.getSeasonalIconId());
```

Comment:
If the »seasonal icon« is the app icon, I agree with this removal. It’s not present in other app using Material Design either.